docs/api/solvers/sde_solvers.md

```diff
@@ -113,3 +113,40 @@ These are reversible in the same way as when applied to ODEs. [See here.](./ode_
     selection:
         members:
             - __init__
+
+
+---
+
+### Underdamped Langevin solvers
+
+These solvers are specifically designed for the Underdamped Langevin diffusion (ULD),
+which takes the form 
+
+$d \mathbf{x}_t = \mathbf{v}_t dt$
+
+$d \mathbf{v}_t = - \gamma \mathbf{v}_t dt - u
+\nabla f( \mathbf{x}_t ) dt + \sqrt{2 \gamma u} d W_t.$
+
+where $\mathbf{x}_t, \mathbf{v}_t \in \mathbb{R}^d$ represent the position
+and velocity, $W$ is a Brownian motion in $\mathbb{R}^d$,
+$f: \mathbb{R}^d \rightarrow \mathbb{R}$ is a potential function, and
+$\gamma , u \in \mathbb{R}^{d \times d}$ are diagonal matrices governing
+the friction and the dampening of the system.
+
+They are more precise for this diffusion than the general-purpose solvers above, but
+cannot be used for any other SDEs. They only accept terms generated by the 
+[`diffrax.make_langevin_term`][] function. They all have the same `__init__` signature.
+For an example of their usage, see the [Langevin example](../../examples/langevin_example.ipynb).
+
+::: diffrax.ALIGN
+    selection:
+        members:
+            - __init__
+
+::: diffrax.ShOULD
+    selection:
+        members: false
+
+::: diffrax.QUICSORT
+    selection:
+        members: false
```

Comment:
For completeness I think I'd prefer to include `__init__` in both of these. (Even though it's the same for all of them.)